src/components/movie/SectionHorizontalScroll.tsx

```diff
@@ -36,7 +36,7 @@ class SectionHorizontalScroll extends React.PureComponent<Props> {
     const { title } = sectionData[sectionKey];
 
     return (
-      <View style={styles.container} accessibilityLabel={title.replace(' ','_')+"_section"}>
+      <View style={styles.container} accessibilityLabel={title.replace(/ /g,'_')+"_section"}>
```

Comment:
```suggestion
      <View style={styles.container} accessibilityLabel={title.replaceAll(' ','_')+"_section"}>
```

Reply:
Apparently RN does not support "replaceAll" function. We leave it as it is.